be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/socialMedia/AddBroadcastChirp.scala

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.pop.model.network.method.message.data.socialMedia
+
+import ch.epfl.pop.json.MessageDataProtocol._
+import ch.epfl.pop.model.network.Parsable
+import ch.epfl.pop.model.network.method.message.data.ActionType.ActionType
+import ch.epfl.pop.model.network.method.message.data.ObjectType.ObjectType
+import ch.epfl.pop.model.network.method.message.data.{ActionType, MessageData, ObjectType}
+import ch.epfl.pop.model.objects.{Hash, Timestamp}
+import spray.json._
+
+case class AddBroadcastChirp(
+                          chirpId: Hash,
```

Comment:
Careful here! this should read `chirp_id`. These fields are the **only** exceptions where we use snake_case in nthe project. The reason behind it is that the library we use for parsing/encoding will look for the fields you defined here in the json payload. In this case, spray-json will not find any `chirpId` field and reject the message because every message you receive will be using the syntax `chrip_id`